### PR TITLE
Wearables Docs Updates

### PR DESCRIPTION
**Pull request by schallock**
* Adds note that publication fee is non-refundable
* Adds note that collection names cannot be changed after publication
* Explains how to add custom thumbnails for wearables/items, and explains that these thumbnails must be 256x256px with transparent backgrounds

### CODE REVIEW

File: _posts/wearables/2021-05-31-publishing-wearables.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ For detailed instructions on how to submit your wearable collection for approval
 6. Once your collection has final approval, you will be notified in the Forum. You will also see a visual indicator in the Wearable Editor next to the approved collection.
 7. With a successful approval, you can begin minting items in your collection!
 
+### Publication fees 
+
+There is a required fee for publishing items. This fee exists to deter users from publishing an excessive number of wearables in an attempt to “spam” the wearables market.
+
+**The fee is a flat rate of 500 MANA per item (not NFT!) in your collection.**
```
Comment (nachomazzara) [inline _posts/wearables/2021-05-31-publishing-wearables.md:27]: I would link to the DAO proposal

File: _posts/wearables/2021-05-31-publishing-wearables.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ For detailed instructions on how to submit your wearable collection for approval
...
+
+For example, if you publish a collection with two items, you will have to pay a fee of 1000 MANA (500 MANA for each item) regardless of the rarity (or how many NFTs can be minted) of those items.
+
+These fees are transferred to the Decentraland DAO, where they are used to help fund the growth of the platform through grants and other initiatives voted on by the greater Decentraland community.
+
+> **Note:** Currently, due to the time and resources required to review each collection submitted, **the 500 MANA publication fee is non-refundable**. If your collection is rejected and cannot be published, you will not receive your 500 MANA back.
```
Comment (nachomazzara) [inline _posts/wearables/2021-05-31-publishing-wearables.md]: It is already `published`. I would remove `and cannot be published`. 

Also, I would add that they have the chance to fix it.

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md [hunk 3]
```diff
@@ -46,6 +46,8 @@ You can select which body shape your wearable is modeled for. The shape can be e
 **Name**  
 The name you would like to give your wearable.
 
+> The name of your collection cannot be changed after publication!
```
Comment (nachomazzara) [inline _posts/wearables/2021-05-31-wearables-editor-user-guide.md:51]: What do you think of moving this to the collection creation section?

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md [hunk 4]
```diff
@@ -156,6 +165,8 @@ For example, if you publish a collection with two items, you will have to pay a
 
 These fees are transferred to the Decentraland DAO, where they are used to help fund the growth of the platform through grants and other initiatives voted on by the greater Decentraland community.
 
+_Currently, due to the time and resources required to review each collection submitted, the 500 MANA publication fee is non-refundable. If your collection is rejected and cannot be published, you will not receive your 500 MANA back._
```
Comment (nachomazzara) [inline _posts/wearables/2021-05-31-wearables-editor-user-guide.md]: same as above related to `cannot be published`

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Thanks @schallock

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md (hunk 4 above)
Comment (ShibuyaMartin) [reply _posts/wearables/2021-05-31-wearables-editor-user-guide.md]: "due to the time and resources required to review each collection submitted, the 500 MANA publication fee is non-refundable" --> is this the only reason for the 500 MANA to not be returned?

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md (hunk 3 above)
Comment (ShibuyaMartin) [reply _posts/wearables/2021-05-31-wearables-editor-user-guide.md:51]: I would put this in the card just before publish as well @nachomazzara . Also saying the same about the rarities.

image (8)

Comment (ShibuyaMartin) [reply _posts/wearables/2021-05-31-wearables-editor-user-guide.md:51]: and maybe about the 500 MANA not be returned

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md (hunk 4 above)
Comment (schallock) [reply _posts/wearables/2021-05-31-wearables-editor-user-guide.md]: That's my understanding, yes. It's not that it's technically impossible to return, but the potential to receive the fee back would defeat the purpose of the fee in the first place.

File: _posts/wearables/2021-05-31-wearables-editor-user-guide.md (hunk 3 above)
Comment (schallock) [reply _posts/wearables/2021-05-31-wearables-editor-user-guide.md:51]: Gotcha. Yes those two points we can add to the confirmation window. I'll work with @nachomazzara on this.

**Pull request merged**